numba_cuda/numba/cuda/target.py

```diff
@@ -67,6 +67,7 @@ def resolve_value_type(self, val):
 class CUDATargetContext(BaseContext):
     implement_powi_as_math_call = True
     strict_alignment = True
+    enable_nrt = True
```

Comment:
This should be opt-in for now with a numba configuration option

Reply:
Which I assume a separate CI item should be introduced to separately test NRT enabled and disabled?